src/Controls/samples/Controls.Sample/Startup.cs

```diff
@@ -209,7 +209,7 @@ public static MauiApp CreateMauiApp()
 #elif WINDOWS
 					// Log everything in this one
 					events.AddWindows(windows => windows
-						.OnNativeMessage((a, b) => LogEvent(nameof(WindowsLifecycle.OnNativeMessage)))
+						//.OnNativeMessage((a, b) => LogEvent(nameof(WindowsLifecycle.OnNativeMessage)))
```

Comment:
Did you mean to leave this commented out?

Reply:
yep, that does a spam of logs to the console and makes windows run slow. Dragging the window is like 100k messages.